tests/e2e-tests/src/orchestration.test.ts

```diff
@@ -100,12 +101,20 @@ describe('orchestration', () => {
     const result = await orchestrationToolCalling();
     expect(result.getFinishReason()).toBe('tool_calls');
 
-    const tool_calls =
-      result.data.orchestration_result.choices[0].message.tool_calls;
+    const assistantMessage = result.getAssistantMessage();
 
-    expect(tool_calls).toHaveLength(1);
-    expect(tool_calls[0].function.name).toBeDefined();
-    expect(tool_calls[0].function.arguments).toBeDefined();
+    if (assistantMessage.tool_calls) {
```

Comment:
[req] `expect(...).toBeDefined()`

But I don't think you need to check all those things this thoroughly. Just `expect(assistantMessage.tool_calls[0].function.name).toBeDefined();` or for `arguments` as well should already be fine. If anything undefined or array is empty, we get error anyways.